src/game/spawn/level.rs

```diff
@@ -0,0 +1,18 @@
+//! Spawn our main level by triggering other spawn events.
```

Comment:
```suggestion
//! Spawn the main level.
```

Reply:
I want to make it explicit that users should not hardcode spawning logic in there and instead defer to other observers.

Reply:
Changed the phrasing a bit [`d1b5de8` (#70)](https://github.com/TheBevyFlock/bevy-template/pull/70/commits/d1b5de82b563baf6d240307f3fbdc22e2e115238)

Reply:
Okay, fair enough.